runtime/manual/basics/vendoring/index.md

```diff
@@ -0,0 +1,34 @@
+# Caching dependencies locally (Vendoring)
+
+If your project has external dependencies, you may want to store them locally to
+avoid downloading them from the internet every time you build your project. This
+is especially useful when building your project on a CI server or in a Docker
+container.
```

Comment:
It's also useful when someone wants to patch or make modifications to remote dependencies.